### PR TITLE
Added chcon for /generated folder

### PR DESCRIPTION
**Pull request by grant050**
Duplicated the 2.1 version (2.2 version was just a sym-link to 2.1) and added an additional row to the chcon list, adding <your Magento install dir>/generated
With thanks to Zefiryn on stackexchange for the suggestion: https://magento.stackexchange.com/questions/205875/cant-create-directory-magento-2-2-1

## This PR is a:

- [ ] New topic
- [ X ] Content update
- [ ] Content fix or rewrite
- [ X ] Bug fix or improvement

## Summary

When this pull request is merged, it will add a missing piece of the selinux security changes required for magento 2.2.
This adds an additional row to the chcon list, adding <your Magento install dir>/generated

## Additional information

List all affected URLs 
- https://devdocs.magento.com/guides/v2.2/install-gde/prereq/security.html

selinux will prevent Magento from running without this additional change.
The error I was seeing in the exception log:
tail /opt/magento/public_html/var/log/exception.log
[2018-11-12 05:25:37] main.CRITICAL: Can't create directory /opt/magento/public_html/generated/code/Magento/Framework/App/Http/.
Class Magento\Framework\App\Http\Interceptor generation error: The requested class did not generate properly, because the 'generated' directory permission is read-only. If --- after running the 'bin/magento setup:di:compile' CLI command when the 'generated' directory permission is set to write --- the requested class did not generate properly, then you must add the generated class object to the signature of the related construct method, only. {"exception":"[object] (RuntimeException(code: 0): Can't create directory /opt/magento/public_html/generated/code/Magento/Framework/App/Http/.
Class Magento\\Framework\\App\\Http\\Interceptor generation error: The requested class did not generate properly, because the 'generated' directory permission is read-only. If --- after running the 'bin/magento setup:di:compile' CLI command when the 'generated' directory permission is set to write --- the requested class did not generate properly, then you must add the generated class object to the signature of the related construct method, only. at /opt/magento/public_html/vendor/magento/framework/Code/Generator.php:135)"} []

whatsnew
Updated the [instructions for working with SELinux](https://devdocs.magento.com/guides/v2.2/install-gde/prereq/security.html) and Magento 2.2.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3287) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3287) before we can accept your contribution.<br/>

**Comment by magento-engcom-team** [comment]
@grant050 thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: guides/v2.2/install-gde/prereq/security.md [hunk 1]
```diff
@@ -1 +1,66 @@
-../../../v2.1/install-gde/prereq/security.md
\ No newline at end of file
+---
+group: installation-guide
+subgroup: Prerequisites
```
Comment (jeff-matthews) [inline guides/v2.2/install-gde/prereq/security.md]: The following frontmatter entries are obsolete and should be deleted:

- `subgroup`
- `menu_title`
- `menu_order`

File: guides/v2.2/install-gde/prereq/security.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+
+### Suggestion for installing the Magento software with Apache
+
+If you choose to enable SELinux, you might have issues running the installer unless you change the *security context* of some directories as follows:
+
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/app/etc
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/var
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/pub/media
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/pub/static
+  chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/generated
```
Comment (jeff-matthews) [inline guides/v2.2/install-gde/prereq/security.md]: The new line that you added for the `generated/` directory needs to be indented to match the preceding lines

File: guides/v2.2/install-gde/prereq/security.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+	Ubuntu: `setsebool -P apache2_can_network_connect=1`
+
+## Opening Ports In Your Firewall {#install-iptables}
+
+Depending on your security requirements, you might find it necessary to open port 80 and other ports in your firewall. Because of the sensitive nature of networking security, Magento strongly recommends you consult with your IT department before proceeding. Following are some suggested references:
+
+*	Ubuntu: [Ubuntu documentation page](https://help.ubuntu.com/community/IptablesHowTo){:target="_blank"}.
+*	CentOS: [CentOS how-to](http://wiki.centos.org/HowTos/Network/IPTables){:target="_blank"} and [CentOS reference page](http://www.centos.org/docs/4/4.5/Security_Guide/s1-firewall-ipt-basic.html){:target="_blank"}.
+
+#### Related topics:
```
Comment (jeff-matthews) [inline guides/v2.2/install-gde/prereq/security.md]: Please delete the **Related topics:** section. We don't really need these links.

**Comment by magento-cicd2** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**